src/PresentationalComponents/Input/Input.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classnames from 'classnames';
+
+const checkTypes = ['checkbox', 'radio'];
+
+const Input = ({type, className, ...props}) => {
+  const classes = checkTypes.hasOwnProperty(type) ? 'pf-c-check' : 'pf-c-form-control';
+  return (
+    <input {...props}
+      type={type}
+      className={classnames(classes, className)}
+    />
```

Comment:
These really should be paired with labels for accessibility
```HTML
<input id="foo" type="checkbox" value={children}/>
<label for="foo">{children}</label>
```

Reply:
This should also apply for `type='text'`. There should never be an input without a label, even if using a placeholder.

Reply:
With this you should be able to do something like
```JSX
import React from 'react';
import { Input } from '@redhat-insights/insights-frontend-components';

const yourCmp = () => (
  <label>Foo</label>
  <Input />
);
```

But sure labeled Input could be exported component as well, so person using this will have option to use input with label as easy as possible.

Reply:
Oh I see, mhhh... well I guess we can provide two components labeled input (mostly aplicable to checkboxes and radio inputs) and regular input. And if consumers want to use checkbox without label it's up to them.

Reply:
@ryelo last changes to this are that when using `Input` component consumer should set `aria-label`  if not it will default to `type`. So `<Input type="checkox" />` will look like this in HTML
```HTML
<input type="checkbox" aria-label="checkbox" class="pf-c-check">
```

Reply:
The first example is correct. Otherwise, it would need to look like
```HTML
<input type="checkbox" id={props.foo} name={props.foo}>
<label for={props.foo}>{children}</label>
```

Reply:
Makes sence, okay it should be fine now:
```HTML
<input type="checkbox" class="pf-c-check" aria-label="checkbox">
<label class="pf-c-form__label" for="15326125380690.a5cf3ng4qe9">
  <input name="15326125380690.a5cf3ng4qe9" id="15326125380690.a5cf3ng4qe9" type="checkbox" class="pf-c-check"> 
  Some label
</label>
```
Those IDs are autogenerated from time + math random.